app/_data/tables/version_errors_konnect.yml

```diff
@@ -255,6 +255,472 @@ messages:
+      `included_status_codes`, `random_status_code` but Kong Gateway versions <
+      3.1.0.0 do not support these fields. Plugin features that rely on these fields
+      are not working as intended.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1.0.0` or above.
+    DocumentationURL: |
+      [Mocking plugin](/hub/kong-inc/mocking/)
+  - ID: D151
+    Severity: error
+    Description: |
+      For the `oauth2-introspection` plugin, `config.anonymous` field has been set to
+      a string value that is not a UUID, possibly to reference a consumer via
+      username. This feature is not supported in Kong Gateway versions < 3.1. The
+      plugin configuration has been changed to remove the `config.anonymous` field.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1.0.0` or above.
+    DocumentationURL: |
+      [OAuth 2.0 Introspection plugin](/hub/kong-inc/oauth2-introspection/)
+  - ID: D152
+    Severity: error
+    Description: |
+      For the `mtls-auth` plugin, `config.anonymous` field has been set to a string
+      value that is not a UUID, possibly to reference a consumer via username. This
+      feature is not supported in Kong Gateway versions < 3.1. The plugin
+      configuration has been changed to remove the `config.anonymous` field.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1.0.0` or above.
+    DocumentationURL: |
+      [mTLS Authentication plugin](/hub/kong-inc/mtls-auth/)
+  - ID: D153
+    Severity: error
+    Description: |
+      For the `ldap-auth-advanced` plugin, `config.anonymous` field has been set to a
+      string value that is not a UUID, possibly to reference a consumer via username.
+      This feature is not supported in Kong Gateway versions < 3.1. The plugin
+      configuration has been changed to remove the `config.anonymous` field.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1.0.0` or above.
+    DocumentationURL: |
+      [LDAP Authentication Advanced plugin](/hub/kong-inc/ldap-auth-advanced/)
+  - ID: D154
+    Severity: error
+    Description: |
+      For the `request-transformer-advanced` plugin, one or more of the following
+      `config` fields are set: `dots_in_keys`, `replace.json_types`,
+      `append.json_types`, `add.json_types` but Kong Gateway versions < 3.1.0.0 do not
+      support these fields. Plugin features that rely on these fields are not working
+      as intended.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1.0.0` or above.
+    DocumentationURL: |
+      [Request Transformer Advanced plugin](/hub/kong-inc/request-transformer-advanced/)
+  - ID: D155
+    Severity: error
+    Description: |
+      Plugin `datadog-tracing` is not available in Kong Gateway versions < 3.2.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.2` or above.
+    DocumentationURL: |
+      [Datadog Tracing plugin](/hub/kong-inc/datadog-tracing/)
```

Comment:
This endpoint doesn't exist. Where should it point?

Reply:
It doesn't exist yet, but will once the 3.2 doc is published. This is correct.